caching-service/src/main/java/org/zowe/apiml/caching/config/GeneralConfig.java

```diff
@@ -20,4 +20,6 @@
 public class GeneralConfig {
     @Value("${caching.storage.evictionStrategy:reject}")
     private String evictionStrategy;
+    @Value("${caching.storage.size:100}")
```

Comment:
Is 100 records a reasonable default?

Reply:
We can probably increase it, maybe 1000?

Reply:
Will be done on another PR